.ado/templates/fluentui-apple-publish-nuget-job-xcode12-beta.yml

```diff
@@ -0,0 +1,39 @@
+jobs:
+- job: fluentui_apple_publish_nuget_xcode12_beta
+  pool:
+    vmImage: 'macos-10.15'
+  displayName: FluentUI Apple Publish NuGet Xcode 12 Beta
+  timeoutInMinutes: 60 # how long to run the job before automatically cancelling
+  cancelTimeoutInMinutes: 5 # how much time to give 'run always even if cancelled tasks' before killing them
+  
+  steps:
+  # Select Xcode 12 beta version
+  - template: apple-xcode-select.yml
+    parameters:
+      xcode_path_override: '/Applications/Xcode_12_beta.app/Contents/Developer'
```

Comment:
Is this the universal beta or the non-universal one? Is there a way to actually distinguish until beta 3 makes it moot?

Reply:
This is the non-universal version. While it won't actually start producing universal binaries, it will set us up to do so once beta 3 comes out.

We are blocked by https://github.com/actions/virtual-environments/issues/1166 on actually building/publishing universal binaries, but that doesn't block us from setting up a parallel publishing pipeline.